test/schemas.ts

```diff
@@ -379,7 +379,7 @@ export const v7Schema = {
   },
   send: {
     ...v6Schema.send,
-    inviteRewardsEnabled: false,
+    inviteRewardsVersion: 'disabled',
```

Comment:
we shouldn't need to change previous migrations?

Reply:
ah, I just ran a grep for anywhere that used `inviteRewardsEnabled`, wasn't sure if this needed to be changed or not. I will change it back.

Reply:
I get this error when I change this back to `inviteRewardsEnabled` (and remove `inviteRewardsVersion`):
```
 FAIL  test/schema.test.ts
  ● getLatestSchema › validates against the RootState schema

    expect(received).toBeNull()

    Received: [{"instancePath": "/send", "keyword": "required", "message": "must have required property 'inviteRewardsVersion'", "params": {"missingProperty": "inviteRewardsVersion"}, "schemaPath": "#/required"}, {"instancePath": "/send", "keyword": "additionalProperties", "message": "must NOT have additional properties", "params": {"additionalProperty": "inviteRewardsEnabled"}, "schemaPath": "#/additionalProperties"}]
```

Reply:
ah, on closer inspection, this is because you haven't done a redux migration. https://github.com/valora-inc/wallet/pull/3454 this is an example of the code we need to add to add a new redux state variable. we're renaming a redux variable in this PR so it is an addition + deletion. docs on why / how to do this are here https://github.com/valora-inc/wallet/blob/main/WALLET.md#redux-state-migration lmk if you have any questions